src/frontend/komponenter/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -16,6 +17,20 @@ import { Behandling } from './Behandling';
 import { BehandlingEllerTilbakekreving } from './BehandlingEllerTilbakekreving';
 import type { VisningBehandling } from './visningBehandling';
 
+const SwitchHøyre = styled(Switch)`
```

Comment:
Litt usikker på hvorfor den heter Høyre, er det fordi man drar switchen kun til høyre? Hadde vært kjekt med et skjermbilde så man får litt mer kontekst 😄

En gang i tiden laget vi en navnkonvensjon-guideline for styled components i team familie for å prøve å holde det mest mulig likt på tvers, den ligger her: https://github.com/navikt/familie 
Den er selvfølgelig åpen for innspill og endringer, men der har vi i allefall prøvd at alle importerte komponenter som wrappes inn i en styled komponent prefixes med Styled

Reply:
Kjempegodt poeng 🤩 

Jeg la til bilder nå. Angående knappen så tror jeg at navnet StyledSwitch skal fungere, hva tenker du @daphneleebeek ? 😊